doc/harmony-ap_logging_user_guide.md

```diff
@@ -0,0 +1,371 @@
+2. **Log capture**: Logback checks the `Logger` log level of the message against its configured minimum log level in the Logback configuration to determine if the message should be processed. If the log level meets or exceeds the configured threshold, the message is processed further.
+3. **Log appending**: Captured log messages are then sent to the appropriate appenders configured in Logback for the `Logger` instance. An appender is a component that determines how logs are formatted, rotated, and delivered to their destinations:
+   - **Log formatting**: The log message is formatted according to the configuration specified in the appender. The formatting adds metadata like timestamps, log levels, thread information, and contextual data to log messages.
+   - **Log rotation**: Logback can optionally manage log rotation by creating new log files when the current log file reaches a certain size or age. This feature helps prevent log files from becoming too large and consuming too much disk space.
+   - **Log output**: Defines how processed log messages are sent to the output destination specified. This could be the console, a file, a remote server, or any other output destination configured in Logback.
+
+![Log lifecycle overview](img/ug_ap_logging_log_lifecycle.svg)
+
+For more information on Logback, refer to the Logback Manual \[[LOGBACK](#Ref_LOGBACK)\].
+
+## 3 Logback configuration in Harmony AP
+
+The configuration of Logback can vary depending on the deployment environment. The following sections describe how logging works in the Linux package and Docker image of Harmony AP.
+
+### 3.1 Logback configuration in Harmony AP Linux package
+
+The default configuration file for Logback is available in the [logback.xml](https://github.com/nordic-institute/harmony-common/blob/main/packaging/ap/config/logback.xml).
+
+#### 3.1.1 Viewing logs
+
+The logs are sent to the standard output or `stdout`, so they can be viewed in the terminal of the host where the AP is running by executing the following command:
+
+```bash
+journalctl -u harmony-ap
+```
+
+Logs are also saved in the following files located in the `/var/log/harmony-ap` directory:
+
+- `security.log`: This log file contains all the security related information. For example, you can find information about the clients who connect to the application.
+- `business.log`: This log file contains all the business-related information. For example, when a message is sent or received, etc.
+- `domibus.log`: This log file contains both the security and business logs plus miscellaneous logs like debug information, logs from one of the framework used by the application, etc.
+- `domibus-error.log`: This log file contains all the errors which occurred in Domibus including errors from third party libraries used by Domibus.
```

Comment:
I think it would be better to talk about Harmony here:

> ...occurred in Domibus including...libraries used by Domibus.
=>
> ...occurred in Harmony including...libraries used by Harmony.